src/busesApp/services.js

```diff
@@ -0,0 +1,36 @@
+import { cleanModel } from '../utils/cleanModel.js';
+import { save } from '../services/saveService.js';
+import Bus from './models.js';
+
+export const registerBus = async (reqBody) => {
+	const newBus = await save(Bus, reqBody);
+	return newBus;
+};
+
+export const findBusByPlate = async (plateNumber) => {
+	const existingBus = await Bus.findOneBy({ plateNumber });
+	return existingBus;
+};
+
+export const findBusById = async (id) => {
+	const bus = await Bus.findOneBy({ id });
+	return bus;
+};
+
+export const findBuses = async () => {
+	const buses = await Bus.find();
+	return buses;
+};
+
+export const updateBus = async (Model, busId, editInfo) => {
+	const bus = await Model.findOneBy({ id: busId });
+	if (!bus) return false;
+	const cleanBusInfo = cleanModel(Model.create(editInfo));
+
+	if (cleanBusInfo) await Model.update(bus, cleanBusInfo);
```

Comment:
I think you don't need to cleanInfo again as you already have matched data in the `handlePutController`